Development-Guide/CAN-Filter-Guide/README.adoc

```diff
@@ -0,0 +1,68 @@
+= CAN Message Filter Guide
+
+== Introduction
+This is a basic guide on how to make a CAN filter.
+The two main components of a filter are an ID and a Mask.
+The ID will be the unique identifier for the message you are looking for, and the Mask will determine which bits of the ID are important to sift through (this makes the proccess much more efficient and flexible). 
+
+IMPORTANT: A Mask must be able to adapt to future additions to a CAN network. It must be made *as tight as possible for longevity*.
+
+There is no set order to follow the steps, but to get started follow the steps below until you are comfortable.
+Another tip is to convert the hex format into binary format, determine the ID and Mask, and then convert those back into hex.
+
+== Steps
+
+. Determine the ID
```

Comment:
I think overall instead of steps the whole thing would look nicer if "Determining the ID" was a header itself. Same with determining the mask. Also if you look at the compiled asciidoc, your bullet points are all messed up.